pkg/cmd/template/cmd_data_values_test.go

```diff
@@ -442,7 +442,8 @@ str: str`)
 
 	out := opts.RunWithFiles(cmdtpl.Input{Files: filesToProcess}, ui)
 
-	expectedErrorMessage := "Expected YAML document to be annotated with data/values but was *yamlmeta.MapItem"
+	expectedErrorMessage := "Found @data/values on Map Item (data.yml:4); only Documents (---) can be annotated with @data/values"
+
```

Comment:
```suggestion
	expectedErrorMessage := "Found @data/values on map item (data.yml:4); only documents (---) can be annotated with @data/values"

```

Let's make node types lowercase as they are ordinary (not proper) nouns.

Reply:
Thank you. I have made the changes.